packages/element-lib/src/sidetree/getNodeInfo.js

```diff
@@ -2,9 +2,9 @@ module.exports = async (sidetree) => {
   // eslint-disable-next-line
   sidetree.getNodeInfo = async () => {
     const accounts = await sidetree.blockchain.web3.eth.getAccounts();
-    const data = await sidetree.storage.ipfs.version();
+    // const data = await sidetree.storage.ipfs.version();
     return {
-      ipfs: data,
+      // ipfs: data,
```

Comment:
why don't we want that?

Reply:
This is one of those things I need to fix... the extension is different if you use a manager instead of a normal storage interface... the coupling here has always bothered me I will fix shortly.

Reply:
See https://github.com/decentralized-identity/element/issues/90

Reply:
fixed